docs/self-managed/web-modeler/installation.md

```diff
@@ -0,0 +1,14 @@
+---
+id: installation
+title: Installation
+---
+
+:::caution Beta Offering
+Web Modeler Self-Managed is currently offered as a [beta release](../../../reference/early-access#beta)
+with limited availability for enterprise customers only. It is not recommended for production use and there is no maintenance service guaranteed.
+Special [terms & conditions](https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/) apply.
+However, we encourage you to provide feedback via your designated support channel or the [Camunda Forum](https://forum.camunda.io/).
+:::
+
+Currently, we only support an installation of Web Modeler Self-Managed with Docker and Docker Compose.
+Please refer to the [Installation Guide](../platform-deployment/docker.md) for more details.
```

Comment:
In both the linked docker guide, as well as here, is missing which env variables are supported. We should have a page and link to it, similar to /docs/next/self-managed/identity/deployment/configuration-variables/